frontend/src/pages/AlertDetails/AlertHeader/AlertIcon/AlertIcon.tsx

```diff
@@ -0,0 +1,49 @@
+import { BellOff, CircleCheck, CircleOff, Flame } from 'lucide-react';
+
+export default function AlertIcon({ state }: { state: string }): JSX.Element {
+	let icon;
+	// TODO(shaheer): implement the states UI based on updated designs after the designs are updated
+
+	switch (state) {
+		case 'no-data':
+		case 'pending':
+			icon = (
+				<CircleOff
+					size={18}
+					fill="var(--bg-sienna-400)"
+					color="var(--bg-sienna-400)"
+				/>
+			);
+			break;
+		case 'muted':
+		case 'disabled':
+		case 'inactive':
+			icon = (
+				<BellOff
+					size={18}
+					fill="var(--bg-vanilla-400)"
+					color="var(--bg-vanilla-400)"
+				/>
+			);
+			break;
+		case 'firing':
+			icon = (
+				<Flame size={18} fill="var(--bg-cherry-500)" color="var(--bg-cherry-500)" />
+			);
+			break;
+		case 'resolved':
+			icon = (
+				<CircleCheck
+					size={18}
+					fill="var(--bg-forest-500)"
+					color="var(--bg-ink-400)"
+				/>
+			);
+			break;
+
+		default:
+			icon = <div />;
```

Comment:
do we need empty div in case of unidentified status ? better to render a fragment here ? Every extra div in the DOM has a significant perf impacts in the web vitals score. 

More reference - https://www.uber.com/en-IN/blog/supercharge-the-way-you-render-large-lists-in-react/?uclick_id=bf8d46e2-756f-4a08-9d5e-9fbc50cd028f